pt/plat.py

```diff
@@ -0,0 +1,22 @@
+import base64
+from Cryptodome.Hash import SHA256
+
+from config import PLAT_CIPHER, PLAT_SIGNER, RANDOM_GENERATOR
+from utils.utils import get_data
+
+TARGETADDRESS = (
+    'WIKETTNX9NVKOGZMOSTXMHFXV9ICADYSVJEPVUPTQQOLEV9YWCSWVPTNF9ETUXEHNJIXZPQWDPDOZ9999'
+)
+
```

Comment:
Maybe you can put this address in environment variable

Reply:
Fix in 530a660.